.baedeker/init_fee.ts

```diff
@@ -0,0 +1,408 @@
+                UnpaidExecution: {
+                  weightLimit: 'Unlimited',
+                },
+              },
+              {
+                Transact: {
+                  originKind: 'Superuser',
+                  requireWeightAtMost: {
+                    refTime: 8000000000,
+                    proofSize: 8000,
+                  },
+                  call: {
+                    encoded:
+                      '0x3501010300451f043205011f00d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d0104',
+                  },
+                },
+              },
+            ],
+          },
+        ),
+      )
+      .signAndSend(alice),
+  );
+
+  console.log('Asset created on AssetHubB');
+  await new Promise(f => setTimeout(f, INTERVAL));
+
+  await retry(() =>
+    apiAssetHubB.tx.foreignAssets
+      .setMetadata(
+        {
+          parents: 1,
+          interior: {
+            X3: [{Parachain: 2001}, {PalletInstance: 50}, {GeneralIndex: 1984}],
+          },
+        },
+        'USDT',
+        'USDT',
+        6,
+      )
+      .signAndSend(alice),
+  );
+
+  console.log('Asset metadata on AssetHubB');
+
+  await new Promise(f => setTimeout(f, INTERVAL));
+
+  await retry(() =>
+    apiRelay.tx.sudo
+      .sudo(
+        apiRelay.tx.xcmPallet.send(
+          {
+            V4: {
+              parents: 0,
+              interior: {
+                X1: [
+                  {
+                    Parachain: 2003,
+                  },
+                ],
+              },
+            },
+          },
+          {
+            V4: [
+              {
+                UnpaidExecution: {
+                  weightLimit: 'Unlimited',
+                },
+              },
+              {
+                Transact: {
+                  originKind: 'Superuser',
+                  requireWeightAtMost: {
+                    refTime: 8000000000,
+                    proofSize: 8000,
+                  },
+                  call: {
+                    encoded:
+                      '0x3501010300451f043205011f00d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d0104',
+                  },
+                },
+              },
+            ],
+          },
+        ),
+      )
+      .signAndSend(alice),
+  );
+
+  console.log('Asset created on AssetHubC');
+
+  await new Promise(f => setTimeout(f, INTERVAL));
+
+  await retry(() =>
+    apiAssetHubC.tx.foreignAssets
+      .setMetadata(
+        {
+          parents: 1,
+          interior: {
+            X3: [{Parachain: 2001}, {PalletInstance: 50}, {GeneralIndex: 1984}],
+          },
+        },
+        'USDT',
+        'USDT',
+        6,
+      )
+      .signAndSend(alice),
+  );
+
+  console.log('Asset metadata on AssetHubC');
+
+  await new Promise(f => setTimeout(f, INTERVAL_ASHB));
+
+  await retry(() =>
+    apiAssetHubA.tx.assets
+      .mint(
+        1984,
+        {id: '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY'},
```

Comment:
```suggestion
        {id: alice.address},
```

Reply:
The same for the similar places

Reply:
yes, I did that, but the script doesn't work correctly after that, so I left it until the tests were fixed.